ArticleTemplates/assets/scss/modules/_campaign.scss

```diff
@@ -0,0 +1,226 @@
+.campaign--snippet {
+    border-top: 1px $garnett-neutral-4 solid;
+    background: $garnett-neutral-8;
+    position: relative;
+    padding: 0 0 6px;
+    margin: 16px 0 36px;
+    font-size: 14px;
+    line-height: 21px;
+
+    &:not([open]) .is-on,
+    &[open] .is-off {
+        display: none;
+    }
+}
+
+/* SUMMARY */
+
+.campaign--snippet > summary {
+    outline: none;
+    padding: 0;
+    border-top: 1px $garnett-neutral-4 solid;
+    list-style: none;
+
+    &::-webkit-details-marker {
+        display: none;
+    }
+
+    .campaign--kicker {
+        display: flex;
+        flex-direction: row;
```

Comment:
Unfortunately older versions of Android (4.3 and below) require `-webkit-prefix` for `flex` and these needed to be added manually in these styled 😞 

`display: flex;`
`display: -webkit-flex;`
`flex-direction: row;`
`-webkit-flex-direction: row;`

Reply:
Ok I'll do that 😭

Reply:
I've just used autoprefixer 😈